LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -1426,7 +1460,10 @@ function getForm(indicatorID, series) {
     dialog.setSaveHandler(function() {
     	let isRequired = $('#required').is(':checked') ? 1 : 0;
         let isSensitive = $('#sensitive').is(':checked') ? 1 : 0;
+
```

Comment:
Add a comment here.  Yes, not needed by anyone currently on the team.  But the two below variables are too close together in spelling.  Call attention to the difference by give a little detail on what they are doing.

Also look into either changing the names or values assigned.  "is_xyz" is boolean.